test/PLCRVotingChallenge.test.js

```diff
@@ -0,0 +1,271 @@
+import lkTestHelpers from 'lk-test-helpers'
+const { expectRevert, expectEvent, expectThrow } = lkTestHelpers(web3)
+const BigNumber = require('bignumber.js');
+
+const RegistryMock = artifacts.require('RegistryMock.sol')
+const Token = artifacts.require('TestToken.sol')
+const PLCRVoting = artifacts.require('PLCRVotingMock.sol')
+const PLCRVotingChallenge = artifacts.require('PLCRVotingChallenge.sol')
+
+contract.only('PLCRVotingChallenge', (accounts) => {
+  let challenge, registry, plcrVoting, token
+
+  const mockAddresses = [
+    '0x4e0100882b427b3be1191c5a7c7e79171b8a24dd',
+    '0x6512df5964f1578a8164ce93a3238f2b11485d1c',
+    '0x687355ca7a320e5420a3db5ae59ef662e4146786'
+  ]
+
+  const CHALLENGER                = accounts[0]
+  const LISTING_OWNER             = accounts[1]
+  const CHALLENGER_STAKE          = 10 * 10 ** 18
+  const COMMIT_STAGE_LENGTH       = 60 * 60 * 24
+  const REVEAL_STAGE_LENGTH       = 60 * 60 * 24
+  const VOTE_QUORUM               = 50
+  const PERCENT_VOTER_REWARD      = 25
+
+  beforeEach(async () => {
```

Comment:
instead of declaring variables outside of the test function blocks, you could do `this.token =`, etc, and use throughout the tests. Just need to change `async () => {` to `async function () {` so that the function has a `this` object for context